Cargo.toml

```diff
@@ -10,7 +10,7 @@ overflow-checks = true
 
 [workspace.dependencies]
 cvlr = "0.4.0" 
```

Comment:
We are already at cvlr `0.4.1` could you do the update alongside? https://crates.io/search?q=cvlr

Should we rather use `0.4` here?